widgetry/src/lib.rs

```diff
@@ -93,6 +94,39 @@ mod backend {
     pub use crate::backend_glow::*;
 }
 
+#[derive(Clone, Copy, Debug)]
+pub enum ControlState {
```

Comment:
I guess this is also used by spinners, dropdowns, persistent splits, etc so `ButtonState` doesn't make sense. "Control" is one of those slightly generic words, but I don't have anything better.